src/main/java/ro/msg/learning/shop/models/Address.java

```diff
@@ -0,0 +1,25 @@
+package ro.msg.learning.shop.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+
+public class Address implements BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "address_id")
+    private int id;
+    private String country;
```

Comment:
Please define columns for the other fields, even if the name is the same as the variable name.